client/src/systems/feeding-system.ts

```
@@ -74,7 +74,7 @@ export function useFeedingSystem(options: FeedingSystemOptions) {
     }, duration);
 
     console.log(`🍽️ Feeding mode started for ${duration / 1000} seconds`);
-  }, []);
+  }, [stopFeeding]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)